crd-generator/src/main/java/io/strimzi/crdgenerator/CrdGenerator.java

```diff
@@ -427,9 +427,16 @@ private void checkClassOverrides(Class<?> crdClass, String methodName, Class<?>.
         }
         result.putAll(properties(crdClass));
         JsonPropertyOrder order = crdClass.getAnnotation(JsonPropertyOrder.class);
+        if (order == null && !isExemptClass(crdClass)) {
+            throw new InvalidCrdException(crdClass.getName() + " missing @JsonPropertyOrder annotation");
```

Comment:
I think you just need to call `err()` (which will log the error and exit at the end, that way you get told about all the errors, rather than having to fix them one at a time).